burn-tensor/src/tensor/api/float.rs

```diff
@@ -251,6 +251,15 @@ where
         Self::new(B::float_set_require_grad(self.primitive, require_grad))
     }
 
+    /// Applies the leaky relu function to the tensor.
+    /// The leaky relu function is defined as `x` if `x > 0` and `alpha * x` otherwise.
+    pub fn leaky_relu(self, alpha: f64) -> Self {
+        Self::new(B::leaky_relu(
+            self.primitive,
+            crate::ElementConversion::elem(alpha),
+        ))
+    }
+
```

Comment:
Activation functions aren't callable on float tensors, so I would remove that function.

Reply:
Done